app/_components/commodities/Form.tsx

```diff
@@ -153,73 +158,93 @@ const Form: React.FC<FormProps> = ({
             </FormErrorMessage>
           </FormControl>
         </Stack>
-        <Stack
-          direction={{ base: 'column', sm: 'column', md: 'column', lg: 'row' }}
-          spacing={4}
-          width="100%"
-          marginTop={4}
-        >
-          <FormControl width="100%">
-            <FormLabel>Include</FormLabel>
-            <StationTypesField register={register} />
-          </FormControl>
-          <FormControl
+        <Collapse in={isExpanded} animateOpacity style={{ width: '100%' }}>
```

Comment:
I don't know if this needs thinking about now or we can leave till later but we should probably show more than just the one field outside of the collapsible section. 

Like station search Id be tempted to show landing pad size because if you're in a conda you don't want to have to remember to open the extra options to change the landing pad size every time (I make this mistake more times than I care to admit haha)

Inara is a good reference of what I mean here. For trade routes they show 4 things, then hide a ton of extra options in the collapsible section (but also show a summary of whats selected when its collapsed

![image](https://github.com/ed-pilots-network/frontend/assets/319883/ccb7aa1c-8046-4b24-922f-1bbcc30cc5f0)

![image](https://github.com/ed-pilots-network/frontend/assets/319883/256fdb28-49dc-4c5d-bcd8-a23cf976b0de)

Reply:
Also keep in mind that by default the forms are expanded. You mentioned making sure you can see landing pad size. That input, as well as all others, are visible at page load currently. They only become hidden when the form is submitted

Reply:
> Inara is a good reference of what I mean here. For trade routes they show 4 things, then hide a ton of extra options in the collapsible section (but also show a summary of whats selected when its collapsed

+1 to just staying close to what Inara is doing. I do like the Edit button switch idea and may be that would be needed as an optimization on mobile due to space constraints but closing the entire search context when submitting is a little off IMO. I would like to see what I have selected when looking at results and in place change things to update results. We should keep important ones always visible, show summary of others, show a button to show more options and then keep everything open that's open when user submitted the search.